ckan/migration/versions/088_delete_extras_which_are_deleted_state.py

```diff
@@ -0,0 +1,14 @@
+# encoding: utf-8
+
+
+def upgrade(migrate_engine):
+    migrate_engine.execute(
+        '''
+            ALTER TABLE "package_extra_revision"
+              DROP CONSTRAINT package_extra_revision_continuity_id_fkey;
```

Comment:
`DROP CONSTRAINT IF EXISTS` is probably a good idea here, I know some people have manually removed these for performance.

Reply:
👍 Done